src/pages/ProductDetail/components/PriceCalculator.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { useState } from 'react';
+import './PriceCalculator.scss';
+
+const PriceCalculator = () => {
+  const [signPrice, setSignPrice] = useState(0);
+  const price = 20;
```

Comment:
```suggestion
  const PRICE = 20;
```
- 앞으로 절대 변하지 않는 상수라면 대문자로 작성해주세요